tsuru/client/plugin.go

```diff
@@ -87,6 +102,19 @@ func installPlugin(pluginName, pluginURL string) error {
 	if err != nil {
 		return err
 	}
+	if strings.Contains(resp.Header.Get("Content-Type"), "application/json") { // If manifest.json URL
```

Comment:
I wouldn't rely on content-type header.
Github does not infer type from the content or extension.

Take this example: https://github.com/tsuru/tsuru-client/releases/latest/download/metadata.json
It is a plain json but has `content-type: application/octet-stream`

I think it is fairly safe to try `json.Unmarshal()` and fallback if errors

Reply:
Adjusted to utilize json.Unmarshal and proceed with processing the manifest if no error :)